python/tvm/relay/op/nn/_nn.py

```diff
@@ -1308,3 +1308,6 @@ def dilate_shape_func(attrs, inputs, _):
 reg.register_shape_func("nn.bias_add", False, elemwise_shape_func)
 reg.register_shape_func("nn.softmax", False, elemwise_shape_func)
 reg.register_shape_func("nn.relu", False, elemwise_shape_func)
+
+
+reg.register_broadcast_schedule("nn.im2col")
```

Comment:
Should not be broadcast if you set `TOpPattern` to `kOpaque`.

Reply:
Shall we change kOpaque to kInjective ? Frankly, I do not understand it. please tell me. Thanks.

Reply:
Let's make it `kInjective` and use `register_injective_schedule` here.

Reply:
Thanks, we have change kOpaque to kInjective, and test passed.